nucypher/cli/main.py

```diff
@@ -341,9 +342,15 @@ def ursula(click_config,
     # Authenticated Configurations
     else:
 
-        # Restore configuration from file
+        # Deserialize network domain name if override passed
+        if network:
+            domain_constant = getattr(constants, network.upper())
```

Comment:
Wait, what?

Reply:
This will execute `constants.__ConstantFactory.__getatter__`